sefaria/views.py

```diff
@@ -210,9 +210,12 @@ def subscribe_sefaria_newsletter(request, email, first_name, last_name):
     body = json.loads(request.body)
     language = body.get("language", "")
     educator = body.get("educator", False)
+    mailing_lists = body.get("lists", [])
```

Comment:
@relyks what happens when lists is empty?

Reply:
Right now, when someone sends an email address to Salesforce that does not already exist, Salesforce automatically subscribes it to the default lists: "Master," "General Updates" (or the one for Hebrew), and "Educator Updates" (if the user is an educator). If the user's email address already exists in Salesforce, the flow in Salesforce does not resubscribe the email address to those lists. When you provide additional newsletter mailing lists, the flow in Salesforce will subscribe the email address to those lists, along with the default ones, if the email address does not already exist. However, if we pass the email address with any of those default newsletter lists as additional ones, Salesforce will resubscribe the email address to those lists.

I will write this behavior as comments for documentation purposes, though I should probably record it elsewhere too.